File: app/src/main/java/com/example/android_kotlin_handson/api/response/RepositoryEntity.java
package com.example.android_kotlin_handson.api.response;

import com.google.gson.annotations.SerializedName;

public class RepositoryEntity {
    @SerializedName("full_name")
    public String fullName;

    @SerializedName("html_url")
    public String htmlUrl;

    @SerializedName("stargazers_count")
    public Integer stargazersCount;

    public RepositoryEntity(String fullName, String htmlUrl, Integer stargazersCount) {
        this.fullName = fullName;
        this.htmlUrl = htmlUrl;
        this.stargazersCount = stargazersCount;
    }
}

File: app/src/main/java/com/example/android_kotlin_handson/api/response/RepositoryEntity.kt
package com.example.android_kotlin_handson.api.response

import com.google.gson.annotations.SerializedName

data class RepositoryEntity(
        @field:SerializedName("full_name") val fullName: String,
        @field:SerializedName("html_url") val htmlUrl: String,
        @field:SerializedName("stargazers_count") val stargazersCount: Int
)